spec/kitchen_spec/directions/bake_chapter_introductions/v2_spec.rb

```diff
@@ -187,7 +187,7 @@
   context 'when book has added targt labels for introduction pages' do
```

Comment:
I'm not familiar with `context` (or much of Ruby), but is "targt" a typo and does that have anything to do with your issue?

Reply:
Yeah it's a typo but the typo ended up being a red herring. I needed to add a second `expect` because the pantry was called twice (once for Ch 1 and once for Ch 2)